packages/api/cms-api/src/dam/files/dam-upload-file.interceptor.ts

```diff
@@ -53,7 +53,10 @@ export function DamUploadFileInterceptor(fieldName: string): Type<NestIntercepto
                     }
 
                     const supportedExtensions = mimedb[file.mimetype]?.extensions;
-                    if (supportedExtensions === undefined || !supportedExtensions.includes(extension)) {
+                    if (
+                        (supportedExtensions === undefined || !supportedExtensions.includes(extension)) &&
+                        !(file.mimetype === "application/x-zip-compressed" && extension === "zip")
```

Comment:
Instead of adding an extra condition in this if statement, I'd create supportedExtensions differenty

```ts
let supportedExtensions: string[] | undefined

if (file.mimetype === "application/x-zip-compressed") {
  // .zip-folders in Windows, not supported by mime-db
 // see <link-to issue>
  supportedExtensions = ["zip"];
} else {
  supportedExtensions = mimedb[file.mimetype]?.extensions;
}

if (supportedExtensions === undefined || !supportedExtensions.includes(extension)) {
...
```

In my opinion this would make the extra if more prominent and understandable